content/guides/models/app/features/panels/line-plot/_index.md

```diff
@@ -15,63 +15,56 @@ Line plots show up by default when you plot metrics over time with **wandb.log()
+      * **Wall Time** is minutes elapsed since the start of the first run on the graph
+      * **Step** increments by default each time `wandb.log()` is called, and is supposed to reflect the number of training steps you've logged from your model
+    * **Y axes**: Select one or more y-axes from the logged values, including metrics and hyperparameters that change over time.
+    * **Min, max, and log scale**: Minimum, maximum, and log scale settings for x axis and y axis in line plots
+    * **Smoothing**: Change the smoothing on the line plot.
+    * **Outliers**: Rescale to exclude outliers from the default plot min and max scale
+    * **Max runs to show**: Show more lines on the line plot at once by increasing this number, which defaults to 10 runs. You'll see the message "Showing first 10 runs" on the top of the chart if there are more than 10 runs available but the chart is constraining the number visible.
+    * **Chart type**: Change between a line plot, an area plot, and a percentage area plot
+  - **Grouping**: Settings for whether and how to group and aggregate runs.
+    * **Group by**: Select a column, and all the runs with the same value in that column will be grouped together.
+    * **Agg**: Aggregation— the value of the line on the graph. The options are mean, median, min, and max of the group.
+  - **Chart**: Titles for the panel, the X-axis, and Ythe -axis, whether or not to show the legend and its position, and font settings for the plot.
+  - **Legend**: Customize the appearance of the panel's legend:
+    * **Legend**: The field in the legend for each line in the plot in the legend of the plot for each line.
+    * **Legend template**: Define a fully customizable template for the legend, specifying exactly what text and variables you want to show up in the template at the top of the line plot as well as the legend that appears when you hover your mouse over the plot.
+  - **Expressions**: Custom calculated expressions to add to the panel:
+    * **Y Axis Expressions**: Add calculated metrics to your graph. You can use any of the logged metrics as well as configuration values like hyperparameters to calculate custom lines.
+    * **X Axis Expressions**: Rescale the x-axis to use calculated values using custom expressions. Useful variables include\*\*_step\*\* for the default x-axis, and the syntax for referencing summary values is `${summary:value}`
+
+1. Click **Apply**.
+
+### All line plots in a section
+To customize the defaults for all line plots in a section, overriding workspace settings for line plots:
+1. Click the section's gear icon to open its settings.
+1. Within the modal that appears, select the **Data** or **Display preferences** tabs to configure the default settings for the section. For details, refer to the preceding section, [Individual line plot]({{< relref "#individual-line-plot" >}}).
```

Comment:
The "display preferences" tab looks like a completely different group of settings that isn't described in the above Individual line plot section... here is what that looks like:
<img width="427" alt="Screenshot 2025-02-27 at 9 39 30 AM" src="https://github.com/user-attachments/assets/5398a8b3-e944-4ae0-a786-974b9ec2598e" />

Reply:
Done